models/istio_validation.go

```diff
@@ -117,6 +117,11 @@ var checkDescriptors = map[string]IstioCheck{
 		Message:  "Namespace not found for this rule",
 		Severity: WarningSeverity,
 	},
+	"authorizationpolicy.source.principalnotfound": {
+		Code:     "KIA0106",
```

Comment:
Side note, this would require a PR in the kiali.io, to have it documented in the reference.

Reply:
https://github.com/kiali/kiali.io/pull/529